sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/indexes/models/SearchIndexerLimits.java

```diff
@@ -49,7 +49,7 @@ public Duration getMaxRunTime() {
      *
      * @return the maxDocumentExtractionSize value.
      */
-    public Double getMaxDocumentExtractionSize() {
+    public Long getMaxDocumentExtractionSize() {
```

Comment:
We'll want to make note of this as a breaking change in the CHANGELOG.

Reply:
Talked offline. This is autorest v4 changes instead of swagger.